.github/workflows/build-and-deploy-website.yml

```diff
@@ -58,9 +58,17 @@ jobs:
       run: |
         rm -rf assets 1>/dev/null 2>&1
         rm -rf cheatsheets 1>/dev/null 2>&1
-        rm -rf gitbook 1>/dev/null 2>&1
-        rm index.html 1>/dev/null 2>&1
-        rm search_index.json 1>/dev/null 2>&1
+        rm -rf search 1>/dev/null 2>&1
```

Comment:
Agree, don't like this very much also. I think what we only want to keep there is CNAME and robots.txt, right ?

Reply:
I have used extglob and the followinfg syntax for the moment t simplify this, oncsidering only CNAME (which I removed from my fork) and robots.txt are the only files to keep :
```bash
shopt -s extglob
rm -rdfv !("CNAME"|"robots.txt")
```

Going to a full github action to publish, and thus limiting the script file would be better I agree but it involves more changes (and tests) I think, especially since we need due to the repo structure to do some "operation" before running mkdocs. Step by Step like you said :smile: